accounts/accounts.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2015 The go-ethereum Authors
```

Comment:
If there's significant changes, the copyright ought to be updated. Is it just a split, or did you change the APIs?